source/navitiacommon/navitiacommon/parser_args_type.py

```diff
@@ -92,16 +92,14 @@ def is_geometry_valid(geometry):
         if value:
             is_valid_object = isinstance(value, dict)
             if not is_valid_object:
+                raise ValueError('invalid json')
+
+            if not is_geometry_valid(value) :
                 raise ValueError('invalid geojson')
-            features= value.get('features')
-            if not features:
-                is_valid_object = is_geometry_valid(value)
-            else:
-                for feature in features:
-                    geometry = feature.get('geometry')
-                    is_valid_object = is_valid_object and is_geometry_valid(geometry)
-            if not is_valid_object:
-                raise ValueError('invalid geometry')
+
+            type= value.get('type')
```

Comment:
hmmm, "type" is a reserved word in python(it's a function), we'd better avoid using it as variable name :)